### ISSUE TITLE
How to handle translation comments/issues?

### ISSUE THREAD

**Issue by aggsol** [liberapay/salon#128]
For example in https://hosted.weblate.org/translate/liberapay/core/de/?checksum=fa8c7e31a2b22f0e there is a IMO valid comment. Should we discuss and resolve it there or create an issue at GitHub?

**Comment by EdOverflow** [comment]
If I may put my two cents in, while quickly skimming over the German translations I noticed numerous issues. Therefore, I would suggest sticking to Weblate otherwise you will end up flooding the GitHub repo with issues.

**Comment by aggsol** [comment]
@EdOverflow Yes, please add your issues regarding the German translation. I am waiting for feedback and especially help.

**Comment by EdOverflow** [comment]
I went ahead and suggested a couple of things.

**Comment by EdOverflow** [comment]
I had a look at German, French and Italian translations and I hope that helps you out a bit.

**Comment by Changaco** [comment]
It's best to report problems with the English source strings on GitHub, either here in the salon or in https://github.com/liberapay/liberapay.com where the code that needs to be improved is.

Discussions about the translated strings should be either on Weblate or here in the salon.
